### PR TITLE
add classname to array list item title

### PR DESCRIPTION
**Pull request by rianfowler**
#320 

Was recommended that we style a class name instead of an h5. 
screen shot 2019-02-19 at 16 01 02

## Types of changes

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I've reviewed the [guidelines for contributing to this repository](../CONTRIBUTING.md#how-to-contribute-to-us-forms-system).
- [x] I've checked style and lint with `npm run lint`.
- [x] I built the production version with `npm run build`.
- [ ] I added tests to verify a bug fix or new functionality.
- [x] All tests pass locally with `npm test`.
- [ ] My change requires a change to the documentation, and I've updated the documentation accordingly.

### CODE REVIEW

File: src/js/review/ArrayField.jsx [hunk 1]
```diff
@@ -211,7 +211,7 @@ class ArrayField extends React.Component {
                     <div className="small-12 columns va-growable-expanded">
                       {isLast
                         // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-                        ? <h5 tabIndex="0" ref={this.handleNewItemTitleMounted}>New {uiOptions.itemName || 'Item'}</h5>
+                        ? <h5 className="schemaform-array-row-title" tabIndex="0" ref={this.handleNewItemTitleMounted}>New {uiOptions.itemName || 'Item'}</h5>
```
Comment (rianfowler) [inline src/js/review/ArrayField.jsx]: @jbalboni I thought schemaform fits in with what's already here but we only style stuff called `form-` so maybe this hsould be `form-review-array-row-title` or `-item-title`?

Comment (jbalboni) [reply src/js/review/ArrayField.jsx]: I think you can still use `schemaform`; we do style classes with that prefix, the styles for that are just in vets-website.

Comment (rianfowler) [reply src/js/review/ArrayField.jsx]: Thanks I've merged the style change into formation. This should be ready to go.

Comment (dmethvin-gov) [reply src/js/review/ArrayField.jsx]: We should talk about how to migrate this codebase back into the VA site. The classes being applied here should technically coordinate with the CSS that the USFS library ships and I don't think that's happening lately. Things are getting further out of sync which will get confusing for anyone who wants to pick up USFS and change it later.

Comment (rianfowler) [reply src/js/review/ArrayField.jsx]: I think once we move the us-forms-system back into `vets-website` we can delete the `1.x-stable` branch

**Pull request closed without merging**